src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -551,13 +584,14 @@ Domain<3> BinaryCompactObject::create_domain() const {
                            {17, Direction<3>::lower_zeta()}}});
   }
 
-  Domain<3> domain{
-      std::move(maps),
-      corners_for_biradially_layered_domains(2, 3, not object_A_.is_excised(),
-                                             not object_B_.is_excised()),
-      {},
-      std::move(boundary_conditions_all_blocks),
-      std::move(excision_spheres)};
+  const size_t num_biradial_layers = need_cube_to_sphere_transition_ ? 3 : 2;
```

Comment:
Can I ask why there are two biradial layers (layers of blocks that surround both black holes)? I thought the layer of frustums is the only layer in this configuration, unless the outermost logarithmic layer is always present?

Reply:
The two biradial layers are (1) frustums transitioning to a sphere, and (2) outer shell that can go to large distance. We can certainly try to remove the outer shell when we don't need to extract at large distance, e.g. with CCM. I don't think we have the information at this time to decide that.